README.md

```diff
@@ -25,6 +25,10 @@ Diff JSON and JSON-like structures in Python.
 >>> diff({'a': [0, {'b': 4}, 1]}, {'a': [0, {'b': 5}, 1]})
 {'a': {1: {'b': 5}}}
 
+# You can exclude some jsonpaths from the diff (doesn't work if the value types are different)
```

Comment:
> doesn't work if the value types are different

I'm not quite understanding this. Does this mean the value type as in the dict key-value? I understand this only works for dict inputs.

Reply:
Well that's really cool! Thank you for explaining.